src/plugins/image/display/image-display.jsx

```diff
@@ -18,17 +23,22 @@ function ImageDisplay({ content, clientSettings }) {
   }
 
   return (
-    <div className="Image">
-      <img className={`Image-img u-max-width-${content.maxWidth || 100}`} src={src} />
+    <div>
+      <div className="Image">
+        <img className={`Image-img u-max-width-${content.maxWidth || 100}`} src={src} />
+      </div>
+      <div className="Markdown CopyrightInfo" dangerouslySetInnerHTML={{ __html: html }} />
```

Comment:
Everything is fine except these lines: The component should have class `Image` (so leave the `className` on the **outer** `div`, the inner `div` is not necessary, actually), the copyright info is a part of the image component, which means, the class should have the syntax `{ComponentName}-{partName}` (for example: `Image-copyrightInfo`)